lib/streamlit/state/widgets.py

```diff
@@ -182,59 +218,102 @@ def coalesce_widget_states(
     return coalesced
 
 
-class WidgetManager:
+class WidgetManager(object):
```

Comment:
I feel like leaving the inheritance from object implicit makes code less noisy

Reply:
:+1: